### ISSUE TITLE
Nvidia Shield Compatibility

### ISSUE THREAD

**Issue by ebenegui** [diasurgical/devilutionX#3506]
I have two devices, and according to the play store, one have support and the other doesn't, but I've tested the app on both devices and both plays perfectly fine.

- A Samsung Galaxy Tab (supported)
- And the Nvidia Shield Tablet [2017 Model] (not supported)

So, I have the game installed on both devices and both plays as expected, I didn't notice any difference in terms of performance. 
But on the nvidia shield, happens if I launch the game and then close it, and re launch it, it won't launch, but if I retry, It launch perfectly fine, this repeats indefinitely. Really strange.

I've tested this a little further and I find that If I force close the app using the android setting, after quit the game, it'll launch fine every time.

I know that the play store says that the game doesn't have support, but the game runs and plays amazing on the shield and using a TV, it also looks really nice with the widescreen support, the gaming experience is also much comfortable using a gamepad so I'm only seeing advantages here.

**Comment by Bulathsinhala** [comment]
Download the “NVIDIA GeForce NOW” (https://play.google.com/store/apps/details?id=com.nvidia.geforcenow) and try, if the apk you downloaded was built to be uploaded to playstore it will try to connect the playstore server since it can’t through your device it may bug out.

**Comment by ebenegui** [comment]
> Download the “NVIDIA GeForce NOW” (https://play.google.com/store/apps/details?id=com.nvidia.geforcenow) and try, if the apk you downloaded was built to be uploaded to playstore it will try to connect the playstore server since it can’t through your device it may bug out.

**Comment by ebenegui** [comment]
> Download the “NVIDIA GeForce NOW” (https://play.google.com/store/apps/details?id=com.nvidia.geforcenow) and try, if the apk you downloaded was built to be uploaded to playstore it will try to connect the playstore server since it can’t through your device it may bug out.

thanks for the reply, but I think that if that will be the case, then it the launch problem should be always, and this happens every second time that I launch it.
Also, the Nvidia geForce app is already installed by default, so I already have it on the device.
I think that the option to log in with google play at start is an option that needs to coded by the devs, and this doesn't appear to be the case.

**Comment by AJenbo** [comment]
> one have support and the other doesn't

We have set the game to be compatible with all devices. I don't see a way for us to more explicitly mark it as compatible with your device. So unless you have some information regarding how to do so I don't think there is anything we can do here. Maybe it is something Nvidia determines?

> I think that the option to log in with google play at start is an option that needs to coded by the devs

I don't see any reason we should link with Google Play?

Closing for now on less you have some additional relevant information.
